controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -135,11 +139,50 @@ func (r *StorageConsumerReconciler) initReconciler(request reconcile.Request) {
 
 func (r *StorageConsumerReconciler) reconcilePhases() (reconcile.Result, error) {
 
+	secret := &corev1.Secret{}
+	secret.Name = r.storageConsumer.Name
```

Comment:
Should we add an annotation to storageConsumer that points to the onboarding ticket secret for that conusmer?

Reply:
the intention was to have the same name for consumer & secret, didn't get your suggestion.

Reply:
changed to have predictable naming.